src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,43 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService = (ManufacturerService) injector
+                .getInstance(ManufacturerService.class);
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        Manufacturer bmw = new Manufacturer("BMW", "Germany");
+        manufacturerService.create(bmw);
+        List<Driver> drivers = new ArrayList<>();
+        Driver mia = new Driver("Mia", "12345");
+        Driver misha = new Driver("Misha", "567890");
+        driverService.create(mia);
+        driverService.create(misha);
```

Comment:
```suggestion
        mia = driverService.create(mia);
        misha = driverService.create(misha);
```
если мы этого не сделаем, то mia и misha останутся без айди, которое мы получили из resultSet